### ISSUE TITLE
[Phase 4] 베이스라인 Fine-tuning 학습 — 대조군 Student(FT)

### ISSUE THREAD

**Issue by YOONPYOGitHub** [YOONPYOGitHub/knowledge-distillation#48]
## 목표
4-Way 비교의 대조군인 Student(FT) 모델 학습 — KD 없이 CE Loss만으로 학습

## 기간
Phase 4 (5~7개월차) | KD 학습과 동시 또는 직후 | 예상 1~2주

## 핵심 — 공정 비교를 위한 통제 변인
> KD Loss만 다르고 나머지 **모든 조건** 동일해야 함

| 변인 | Student(KD) | Student(FT) |
|------|-------------|-------------|
| 기본 모델 | gpt2 (124M) | gpt2 (124M) ✅ 동일 |
| 학습 데이터 | WikiText-2 | WikiText-2 ✅ 동일 |
| Epochs | 3~5 | 3~5 ✅ 동일 |
| Learning Rate | 5e-5 | 5e-5 ✅ 동일 |
| Batch Size | 8 | 8 ✅ 동일 |
| Optimizer | AdamW | AdamW ✅ 동일 |
| **Loss 함수** | **KD + CE** | **CE만** ❌ 유일한 차이 |

## 구현 항목
- [ ] CE Loss 기반 training loop (Teacher 불필요)
- [ ] 동일 하이퍼파라미터 적용 검증
- [ ] 동일 random seed 사용 (재현성)
- [ ] Validation + Checkpoint 저장
- [ ] Full Run (3~5 epochs)

## 완료 기준
- [ ] Student(FT) 학습 완료, checkpoint 저장
- [ ] KD 학습과 동일 조건 준수 확인
